stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/sync/internal/SyncManager.kt

```diff
@@ -245,86 +250,102 @@ internal class SyncManager(
         }
     }
 
-    private suspend fun retryChannels(): List<Channel> {
-        return repos.selectChannelsSyncNeeded().onEach { channel ->
-            chatClient.createChannel(
+    private suspend fun retryChannels() {
+        val cids = repos.selectChannelCidsBySyncNeeded()
+        logger.logD("[retryChannels] cids.size: ${cids.size}")
+        cids.forEach { cid ->
+            logger.logD("[retryReactions] process channel($cid)")
+            val channel = repos.selectChannelByCid(cid) ?: return@forEach
+            logger.logV("[retryChannels] sending channel($cid)")
+            val result = chatClient.createChannel(
                 channel.type,
                 channel.id,
                 channel.members.map(UserEntity::getUserId),
                 channel.extraData
             ).await()
+            logger.logV("[retryChannels] result($cid).isSuccess: ${result.isSuccess}")
         }
     }
 
     @VisibleForTesting
-    private suspend fun retryMessages(): List<Message> {
-        return retryMessagesWithSyncedAttachments() + retryMessagesWithPendingAttachments()
+    private suspend fun retryMessages() {
+        logger.logD("[retryMessages] no args")
```

Comment:
Do we want to keep those logs?

Reply:
The idea was to provide a way (through the `ChatLoggerHandler`) for  Customers to catch some breadcrumbs according to `CursorWindowAllocationException`.

So customers could use something like Firebase Remote logging in order to get more details rather than just a stacktrace.